kalite/coachreports/static/js/coachreports/coach_reports/views.js

```diff
@@ -79,12 +80,67 @@ var CoachSummaryView = BaseView.extend({
         "click #show_tabular_report": "toggle_tabular_view"
     },
 
+    displayRadialGraph: function(target_elem, data_sub, data_total) {
+        if(!data_sub || !data_total) {
+            document.getElementById(target_elem+"_p").innerHTML = "N/A";
```

Comment:
Better to use jQuery instead of document.getElementById for cross browser compatibility purposes (again, IE doesn't play nice sometimes).